lib/byron/test/data/cardano-node/node.config

```diff
@@ -0,0 +1,73 @@
+# vim: set filetype=yaml:
+# -*- mode: yaml -*-
+
+#     _   _           _          ____             __ _
+#    | \ | | ___   __| | ___    / ___|___  _ __  / _(_) __ _
+#    |  \| |/ _ \ / _` |/ _ \  | |   / _ \| '_ \| |_| |/ _` |
+#    | |\  | (_) | (_| |  __/  | |__| (_) | | | |  _| | (_| |
+#    |_| \_|\___/ \__,_|\___|   \____\___/|_| |_|_| |_|\__, |
+#                                                      |___/
```

Comment:
very nice comments:-)

Reply:
:upside_down_face:

Reply:
One has to find an occupation when re-compiling a new LTS.